### DISCUSSION TITLE
2 questions for Onedrive backup - archive attribute and auto rescan of files

### DISCUSSION THREAD

**Discussion by dagremote** [iterate-ch/cyberduck#14433] [Q&A]
Bought the Duck, no problems with mounting and setup. Everything set to defaults and smart sync.

I am trying to now backup Onedrive and it appears to work for a full backup. Challenges on next backup are that the Archive bit doesn't appear to follow the actual value on any given file. All files have the archive bit disabled, even though the Onedrive archive bit is set. Second item is about getting a scan of the Onedrive files before a backup. I tried to right-click, then rescan, and that either doesn't work or I'm not waiting long enough. Only way that seems to work is to go to the folder (using a mounted Duck drive letter) where I know files were modified and then waiting a bit. Ideas on how to get the files in sync needed. Ok with a CLI option if it's easiest. I can simply schedule an early morning scan of the Onedrive in a scheduled task - as an example.

Using Windows and not Mac.

Thanks.

**Comment by AliveDevil** [comment]
Mountain Duck is not a backup application, "Keep Offline" is a method of keeping remote files locally, so that you don't have to be online to work with those files. You can enable the Indexer to let Mountain Duck regularly check in visited directories for changes. Mountain Duck also refreshes directory listings if there is an active Explorer-window in a mounted directory more often than the indexer is run.

What do you want to achieve with the archive-bit? Remote storages (like OneDrive, FTP, SFTP) don't know anything about NTFS file attributes, that is why Mountain Duck doesn't persist these.

**Reply by dagremote** [reply]
Thanks. I'll test the indexing option. I get it that Mountain Duck in itself is not a backup app. What I'm attempting to do is to use it to present cloud data to a backup app. So, it cloud storage for Onedrive, or Box, or Dropbox gets mounted as a drive letter, they can then be set as the source of a backup job. This is actually working in early tests but as mentioned in my first entry, I'm only getting the initial full backup.

I'll keep digging.
